app/controllers/application_submissions_controller.rb

```diff
@@ -23,16 +23,16 @@ def create
 
   def csv_export
     respond_to :csv
-    start_date = parse_date_picker_param(:start_date)
-    end_date = parse_date_picker_param(:end_date)
+    start_date = Date.parse params[:start_date]
+    end_date = Date.parse params[:end_date]
     @records = ApplicationSubmission.in_department(given_or_all_department_ids)
                                     .between(start_date, end_date)
```

Comment:
Not requesting changes be made here, but eventually I think it would be nice to move any parsing to `.between`, keeping in line with "cast in the model layer".

Also goes for the other class methods below.

Reply:
:+1: `.eeo_data` actually just passes those dates to `.between`, so pretty easy.

Reply:
Yeah `&.to_date` should be pretty unobtrusive as well